testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap640/DomainAdjuster640.java

```diff
@@ -287,6 +287,10 @@ public void adjustInfinispanStatisticsEnabled(final List<ModelNode> list, final
 
     private Collection<? extends ModelNode> replaceUndertowWithWeb(final PathAddress subsystem) {
         final List<ModelNode> list = new ArrayList<>();
+
+        //mod_cluster depends on the listener capability so we need to remove the subsystem before we can remove undertow
+        list.add(createRemoveOperation(subsystem.getParent().append(SUBSYSTEM, "modcluster")));
```

Comment:
Ha! This is the version from before the dicsussion that actually initiated it; since as you say, it is not just test failure but a real loss of support of modcluster subsystem altogether.